src/main/java/chat/voiceup/openfire/plugin/httpfileupload/LambdaSlotService.java

```diff
@@ -14,6 +16,8 @@
 
     private AWSLambda lambdaClient = AWSLambdaClientBuilder.standard().build();
 
+    private static final Logger Log = LoggerFactory.getLogger( LambdaSlotService.class );
```

Comment:
Is that some Java thing to add whitespace around `LambdaSlotService.class`?

Reply:
Not really. I copied this line from elsewhere. I'll fix the formatting.